config/openconext/parameters.yaml.dist

```diff
@@ -0,0 +1,78 @@
+parameters:
+    trusted_proxies:   ~
+
+    app_env: prod
+    app_debug: false
+    app_secret: NotSoSecretReplaceMe!
+
+    mailer_transport:  smtp
+    mailer_host:       mailcatcher
+    mailer_user:       ~
+    mailer_password:   ~
+
+    default_locale:    en_GB
+    locales:           [nl_NL, en_GB]
+    locale_cookie_domain:   example.org
+    secret:            NotSoSecretReplaceMe!
+
+    debug_toolbar:          true
+    debug_redirects:        false
+    use_assetic_controller: true
+
+    gateway_api_url: https://gateway.dev.openconext.local/
+    gateway_api_username: ra
+    gateway_api_password: ra_secret
+
+    middleware_credentials_username: ra
+    middleware_credentials_password: ra_secret
+    middleware_url_command_api: https://middleware.dev.openconext.local/command
+    middleware_url_api: https://middleware.dev.openconext.local/
+
+    sms_originator: SURFStepup
+    sms_otp_expiry_interval: 900 # 15 minutes
+    sms_maximum_otp_requests: 10
+
+    saml_sp_publickey: /config/ra/ra_saml_sp.crt
+    saml_sp_privatekey: /config/ra/ra_saml_sp.key
+    saml_metadata_publickey: /config/ra/ra_saml_sp.crt
+    saml_metadata_privatekey: /config/ra/ra_saml_sp.key
+    saml_remote_idp_entity_id: https://gateway.dev.openconext.local/authentication/metadata
+    saml_remote_idp_sso_url: https://gateway.dev.openconext.local/authentication/single-sign-on
+    saml_remote_idp_certificate: 'MIIDwTCCAqmgAwIBAgIUYuSUugwc4J4NyW9WGqYJ/liwM4owDQYJKoZIhvcNAQELBQAwcDELMAkGA1UEBhMCTkwxEDAOBgNVBAgMB1V0cmVjaHQxEDAOBgNVBAcMB1V0cmVjaHQxJzAlBgNVBAoMHkRldmVsb3BtZW50IERvY2tlciBlbnZpcm9ubWVudDEUMBIGA1UEAwwLR2F0ZXdheSBJRFAwHhcNMjMwNTE3MTIxNTEyWhcNMzMwNTE0MTIxNTEyWjBwMQswCQYDVQQGEwJOTDEQMA4GA1UECAwHVXRyZWNodDEQMA4GA1UEBwwHVXRyZWNodDEnMCUGA1UECgweRGV2ZWxvcG1lbnQgRG9ja2VyIGVudmlyb25tZW50MRQwEgYDVQQDDAtHYXRld2F5IElEUDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAM2ulQVs5WpbJOAf7Cv/VPDTJqbWHVdUxAmdwZJlcNTRKNFVp4aJzQ3dpiyiGghI5odnzU0/BWBoHZFNYPU/OFr/gzn6iJGxL63L9+mFgE8PR9HpkV5TaRnr21+nZ0EXWjDZk9Px0enERicCItTeQzAUJeA0A9miIcK5IKIz/zSBSR3c802SGD/VelUqY7Z2/UJM97cT92L+4Fz+4zhxxoThbPbrR0CweiROIt82grdwg7zf0+b62MOuVtqFh0yPLRAFfLc4LjHuxFUdUvOHVta7x74dwdmHikqfujM10XN+sNns3LDJde2yPWchU6ktq7cjgbYfIW/vzVzafP1Jk40CAwEAAaNTMFEwHQYDVR0OBBYEFGYn6LWRDZa7+YryUncIlwJB2VorMB8GA1UdIwQYMBaAFGYn6LWRDZa7+YryUncIlwJB2VorMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAJ57lcOF6PWWW56mS2s5gKFImtfRFzlfiyHsF14L7+nQ5NjfOhpU0wRpnTjK91KP0wCwlxzGFXR8yfqfBFJryIV7aDdYPH/RIkwVaNBI0fsD/ozlYb18seieDEGLvQtTlrmc0UNHtWz6FW3L2geM3ENaqpOATl1Ywp4EPML7Dh0CbhhyM8PnPCEsdclouIeP5/B9Swfk3omXehof6bkFbntqA03msFBiW50twkfKeKULcJGXo667hto27KNxZUauqtPbnAGpUQmge8nxSQlN8RPwlvygVM4LVMF9qP9YxloTH0xVNwN4noZUhfMNsKoJ7Hg5Xulaok8oCqmzEiSroEg='
+    loa_required_for_login: 'http://dev.openconext.local/assurance/loa3'
+    authentication_context_class_ref: '%loa_required_for_login%'
```

Comment:
This seems to be an new config parameter, and probably does the same as `loa_required_for_login`. Please use the old parameter and drop this, or explain the purpose of this new parameter

Reply:
Fair point. Merged them. And kept `authentication_context_class_ref